ci-operator/step-registry/ipi/conf/vsphere/windows-machineset/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
+- jhou1 
+reviewers:
+- jhou1 
+- sunzhaohua2 
+- huali9
+- miyadav
```

Comment:
Could you add @rrasouli as reviewer please?

Reply:
done